hooks/post_daemon_start/20-image-aliases.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+echo $PLUGIN_IMAGE_ALIASES | jq -r 'to_entries[] | [.key, .value] | @tsv' | while read ORIGINAL NEW; do
```

Comment:
It's been a while since I did drone plugin dev work, does Drone inject the dict variables as JSON formatted text for the env var?

Reply:
Managed to demo the answer to my own question, it does indeed.

Reply:
Yeah, it's weirdly inconsistent about it though:
* for a single value drone parameter, it comes through as a